token_server/README.md

```diff
@@ -34,9 +34,59 @@ unless it is getting traffic.
 [Dashboard](https://console.cloud.google.com/appengine?folder=&organizationId=838088520005&project=mapping-crisis).
 You must be logged in as `gd-earthengine-user@givedirectly.org` to access it.
 
-## Run server on Amazon EC2
+### Set up new Google Cloud instance
+* Most of the necessary work needs to be done to set up the mapping page to work
+properly in the first place. The only additional work is to create a service
+account that is whitelisted for EarthEngine access (currently
+`earthengine-token-provider`) and give it the "Service Account Token Creator" role
+in the
+[IAM page](https://console.developers.google.com/iam-admin/iam?project=mapping-crisis).
 
-`¯\_(ツ)_/¯`
+* If you are serving tokens from Google App Engine, and not from Amazon, then
+you should also give the default AppEngine service account
+(currently `mapping-crisis@appspot.gserviceaccount.com`) the "Service Account
+Token Creator" role, so that it can create tokens for the service account you
+created above.
+
+## Run server on Amazon Elastic Beanstalk
+
+* Deploy using [these instructions](https://aws.amazon.com/getting-started/tutorials/deploy-app-command-line-elastic-beanstalk/).
+Since the project already exists on Amazon Elastic Beanstalk, you just need to
+run `eb deploy` after running `eb init` in order to log in.
+
+* The server URL will be entered here once we have GiveDirectly's account.
+
+* See the [console](https://console.aws.amazon.com/) for more information. We
+are using Elastic Beanstalk.
+
+### Set up new Amazon Elastic Beanstalk instance
+* See the [Google Cloud instructions](#set-up-new-google-cloud-instance) if that is
+also changing.
+* Delete the [.elasticbeanstalk](./.elasticbeanstalk) subdirectory of this directory, and follow the
+instructions [here]((https://aws.amazon.com/getting-started/tutorials/deploy-app-command-line-elastic-beanstalk/)
+to set up from scratch.
+* Delete the `.gitignore` file that setup creates, and add `.elasticbeanstalk/*`
+to git.
+* Add Google service account credentials to
+[Amazon Secrets Manager](https://aws.amazon.com/secrets-manager/getting-started/).
+Remember that you need to be logged into Google as
+`gd-earthengine-user@givedirectly.org`.
+   * The Google service account is currently
+`earthengine-token-provider@mapping-crisis.iam.gserviceaccount.com` (specified
+in [ee_token_creator.js](./ee_token_creator.js).
+   * Download private key JSON for the Google service account via the
+   [Credentials page](https://console.developers.google.com/apis/credentials?project=mapping-crisis).
+   * Upload to the [Amazon Secrets Manager](https://aws.amazon.com/secrets-manager/getting-started/)
+   using the provided instructions.
+To make the secret readable by the server, you will have to have the server run
+as a user with permissions to read the secret:
```

Comment:
is the colon supposed to indicate the next line of instructions? Maybe double indent the next two lines?

Reply:
Yeah, I wasn't doing proper md formatting. Should be better now.